css/style.css

```diff
@@ -0,0 +1,70 @@
+*{
+    margin:0
+  
+}
+body{
+    background-color:rgb(179, 180, 187);
+}
+
+nav {
+    margin: auto;
+    display: block;
+    position: fixed;
+
+}
+.menu {
```

Comment:
Excellent use of html classes!😃
For best readability, remember to leave a blank line after the closing curly bracket and the next html class.

Reply:
Excellent use of html classes!!!😃
It is good practice to leave a blank line after the closing curly bracket and the next html class. This helps with a better understanding of your code, making it easier for your teammates and other programmers to read.